src/elements/content-uploader/ContentUploader.js

```diff
@@ -849,7 +852,10 @@ class ContentUploader extends Component<Props, State> {
         const someUploadHasFailed = items.some(uploadItem => uploadItem.status === STATUS_ERROR);
         const allItemsArePending = !items.some(uploadItem => uploadItem.status !== STATUS_PENDING);
         const noFileIsPendingOrInProgress = items.every(
-            uploadItem => uploadItem.status !== STATUS_PENDING && uploadItem.status !== STATUS_IN_PROGRESS,
+            uploadItem =>
+                (uploadItem.status === STATUS_COMPLETE || uploadItem.status === STATUS_ERROR) &&
+                uploadItem.status !== STATUS_PENDING &&
+                uploadItem.status !== STATUS_IN_PROGRESS,
```

Comment:
Is there a test that should be updated to accommodate this change?

Reply:
I don't think this change affects the function's behavior significantly enough to add a test. Line 856 was added to ensure the upload status is either successful or failed upon completion. There is a test that covers updateViewAndCollection; however, it was not affected by the change and a lot of the functionalities are not covered in the test suite. Please share your thoughts if you feel strongly about the adding a test.